pom.xml

```diff
@@ -288,6 +288,31 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+        <groupId>com.fasterxml.jackson.dataformat</groupId>
+        <artifactId>jackson-dataformat-yaml</artifactId>
+        <version>2.3.0</version>
```

Comment:
please use latest version https://mvnrepository.com/artifact/com.fasterxml.jackson.dataformat/jackson-dataformat-yaml

to fix snyk violations:
![image](https://user-images.githubusercontent.com/812984/86477406-39c23c80-bcfd-11ea-8c7c-2738009233f4.png)

please do the same for all other dependencies.